File: week04-2.py
#week04-2.py 是今天的leetcode 挑戰題easy (二合一)
#leetcode 2529.maximum count of positive integer and negative integer
#找出有幾個正數，有幾個負數，大的那個數回傳
class Solution:
    def maximumCount(self, nums: List[int]) -> int:

        pos=0
        neg=0#迴圈前面，準備好，都是0
        for i in range(len(nums)):#迴圈裡，更新他
            if nums[i]>0: pos+=1
            if nums[i]<0: neg+=1
        #迴圈後面，把答案拿出來用
        return max(pos,neg)

File: week04-2.c

//week04-2.c琌さぱleetcode 珼驹肈easy 肈
//leetcode 2529 maximum count of positive integer and negative integer
int maximumCount(int* nums, int numsSize) {

    int pos=0,neg=0;//癹伴玡常琌0

    for(int i=0;i<numsSize;i++){
        if(nums[i]>0)pos++;//タ计
        if(nums[i]<0)neg++;//璽计
    }//癹伴い丁穝э
    //癹伴рㄓノ
    if(pos>neg)return pos;//タ计耕タ计
    else return neg;//ぃ礛碞璽计

}
